src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
+$calendar-block-width: 100px;
+$calendar-block-height: 100px;
+$calendar-block-gap: 1px;
```

Comment:
create separate files for bem blocks, variables and so on

checklist point
<img width="646" alt="image" src="https://user-images.githubusercontent.com/102411130/223057971-889d9564-ef36-474c-b025-5e2bbc56f18c.png">